policy/scan/local_scanner.go

```diff
@@ -130,6 +131,8 @@ func (s *LocalScanner) Schedule(ctx context.Context, job *Job) (*Empty, error) {
 }
 
 func (s *LocalScanner) Run(ctx context.Context, job *Job) (*ScanResult, error) {
+	s.scanMutex.Lock()
+	defer s.scanMutex.Unlock()
```

Comment:
Maybe we should leave a comment here so that we know in the future why this was necessary? Maybe just a link to this PR?

Reply:
added a comment